README.md

```diff
@@ -9,4 +9,4 @@ in Julia.
 
 **Documentation:** [![Latest](https://img.shields.io/badge/docs-latest-blue.svg)](https://juliastats.github.io/StatsModels.jl/latest)
 
-Originally part of [DataFrames.jl](http://github.com/JuliaStats/DataFrames.jl).
+Originally part of [DataTables.jl](http://github.com/JuliaStats/DataTables.jl).
```

Comment:
It might be worth expanding on this a little in the readme; something along the lines of "this was once part of dataframes, now it depends on datatables, but eventually will support generic tabular data stores including both dataframes and datatables"